### PR TITLE
[feature] Added sticky message #347

### LINKED ISSUE #347: [feature] Add possibility to configure sticky message
Add the possibility to configure a sticky message with custom HTML which is shown in all pages.

It can be used for communicating scheduled maintenance operations or any other important message which the provider of the service wants to communicate to their users.

### PR DESCRIPTION
**Pull request by codesankalp**
Added custom html support for sticky message

Closes #347

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -441,6 +441,25 @@ contact_page:
       </div>
 ```
 
+#### Header
```
Comment (nemesisdesign) [inline README.md:444]: I think this should be called "Sticky message" and should be listed after "Second logo" above.

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
Good progress @codesankalp, I also have a few doubts here.

**Review by nemesisdesign — APPROVED** [review]
I sent some improvemens in a1121d38b0dd151a1c733b083f5c3e4bcd30ce11, please double check and let me know. LGTM now :+1:.

**Review by codesankalp — COMMENTED** [review]
Yes it looks good

**Pull request merged**